src/ext.js

```diff
@@ -1,9 +1,25 @@
 import actions, { checkShowAction, getActionsList } from './utils/actions';
 import { checkShowNavigation, getNavigationsList } from './utils/navigation-actions';
+import { getAutomation } from './utils/automationHelper';
 import propertyResolver from './utils/property-resolver';
 import importProperties from './utils/conversion';
 import luiIcons from './utils/lui-icons';
 
+let automationsList = null;
+
+const getAutomations = async () => {
+  if (!automationsList) {
```

Comment:
I'm guessing this is done so that `getAutomations` only fetches the list of automations once? Any particular reason why do that for `automationId` options but not for the `automation` options?

Reply:
That is correct - I did not do this for the `automation` option as I just kept the old logic as it is (it will be no longer be used if the feature flag `SENSECLIENT_IM_1855_AUTOMATIONS_MULTI_USER` is present)

Reply:
Ok and the fetched data is expected to not became stale during its lifetime?

Reply:
That is correct